aui.views/src/AUI/Platform/ATouchscreenKeyboardPolicy.h

```diff
@@ -0,0 +1,32 @@
+// AUI Framework - Declarative UI toolkit for modern C++20
+// Copyright (C) 2020-2024 Alex2772 and Contributors
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+/**
+ * @brief Determines how change touchscreen keyboard state will be performed
+ */
+enum class ATouchscreenKeyboardPolicy {
```

Comment:
Behaviours of show/hide TouchscreenKeyboard() in regards to `policy` are different. The methods should behave to policy equally (as I expected) or the behaviour should be documented in both methods with both enum values.

At the current stage, behaviour is not obvious and hard to understand. I've already told you about mess of implementation of this feature in Android and iOS, let's not repeat their mistakes.